pkg/trace/api/normalizer.go

```diff
@@ -21,48 +23,81 @@ const (
 	MaxNameLen = 100
 	// MaxTypeLen the maximum length a span type can have
 	MaxTypeLen = 100
-	// MaxEndDateOffset the maximum amount of time in the future we
-	// tolerate for span end dates
-	MaxEndDateOffset = 10 * time.Minute
+	// DefaultServiceName TODO: do we really want such indescriptive defaults here like "service"?
+	DefaultServiceName = "no_name_service"
```

Comment:
@gbbr @palazzem do we really want such generic names as defaults or should it be something like this to make it more obvious in the UI that the name is missing?

Reply:
True we already agreed on it, and these values will probably be fine. Updated to use them https://github.com/DataDog/datadog-agent/pull/3738/commits/872d3d410a1a02efb0f92cce7b7c0dbd3f52919e